doc/onebox.md

```diff
@@ -1,103 +1,124 @@
+1. cd example/onebox/bin
+
+
+## Start & Stop Tera
+* Start Tera
 
-## 启动、停止Tera
-* 执行./launch_tera.sh即可启动Tera
 ```
-[tera@localhost bin]$ sh launch_tera.sh 
+[tera@localhost bin]$ ./launch_tera.sh 
 launching tabletnode 1...
 launching tabletnode 2...
 launching tabletnode 3...
 launching master...
 ```
-* 执行./kill_tera.sh即可停止Tera
 
-## 体验开始！
-如果启动正常，尝试查看集群基本信息: `./teracli show`
+* Stop Tera
+
+```
+[tera@localhost bin]$ ./kill_tera.sh
+```
+
+## Shell Exercises
+
+* List all tables
+
 ```
 [tera@localhost bin]$ ./teracli show
-     tablename   status        size   lg_size  tablet  notready
+	 tablename   status		size   lg_size  tablet  notready
 -----------------------------------------------------------------
-  0  meta_table  kTableEnable  4.01K  4.01K    1       0       
-  1  stat_table  kTableEnable  4K     4K       1       0       
-  -  total       -             8.01K  -        2       0    
+  0  meta_table  kTableEnable  4.01K  4.01K	1	   0	   
+  1  stat_table  kTableEnable  4K	 4K	   1	   0	   
+  -  total	   -			 8.01K  -		2	   0	
 ```
 
-查看更详细的表格信息：`./teracli showx`
+* List all tables with more details
+
 ```
 [tera@localhost bin]$ ./teracli showx
-     tablename   status        size    lg_size  tablet  notready  lread  read  rmax  rspeed  write  wmax  wspeed  scan  smax  sspeed
+	 tablename   status		size	lg_size  tablet  notready  lread  read  rmax  rspeed  write  wmax  wspeed  scan  smax  sspeed
 --------------------------------------------------------------------------------------------------------------------------------------
-  0  meta_table  kTableEnable  4.01K   4.01K    1       0         0      0     0     0B/s    0      0     0B/s    0     0     0B/s  
-  1  stat_table  kTableEnable  28.06K  28.06K   1       0         0      0     0     0B/s    0      0     0B/s    0     0     0B/s  
-  -  total       -             32.07K  -        2       0         0      0     -     0B/s    0      -     0B/s    0     -     0B/s  
+  0  meta_table  kTableEnable  4.01K   4.01K	1	   0		 0	  0	 0	 0B/s	0	  0	 0B/s	0	 0	 0B/s  
+  1  stat_table  kTableEnable  28.06K  28.06K   1	   0		 0	  0	 0	 0B/s	0	  0	 0B/s	0	 0	 0B/s  
+  -  total	   -			 32.07K  -		2	   0		 0	  0	 -	 0B/s	0	  -	 0B/s	0	 -	 0B/s  
 ```
 
-查看当前有哪些存活的tabletnode: `./teracli showts`
+* List all tabletservers
+
 ```
 [tera@localhost bin]$ ./teracli showts
-     address         status  workload  tablet  load  busy
+	 address		 status  workload  tablet  load  busy
 -----------------------------------------------------------
-  0  127.0.0.1:7701  kReady  4.01K     1       0     0   
-  1  127.0.0.1:7702  kReady  32.06K    1       0     0   
-  2  127.0.0.1:7703  kReady  0         0       0     0   
+  0  127.0.0.1:7701  kReady  4.01K	 1	   0	 0   
+  1  127.0.0.1:7702  kReady  32.06K	1	   0	 0   
+  2  127.0.0.1:7703  kReady  0		 0	   0	 0   
 ```
 
-查看更详细的tabletnode信息：`./teracli showtsx`
+* List all TabletServers with more details
+
```

Comment:
"List all TabletServers with more details" -> "List all tablet servers with more details", is better?